sources/packages/backend/libs/services/src/workflow/workflow-client.service.ts

```diff
@@ -32,7 +32,7 @@ export class WorkflowClientService {
         `Error while starting application assessment workflow: ${workflowName}`,
       );
       this.logger.error(error);
-      // The error is not thrown here, as we are failing silently.
+      throw error;
```

Comment:
Can you please brief, why we are moving out from the existing fail silent scenario?

Reply:
same qn

Reply:
The fail silent was in place because the workflow actions service was invoked in the same thread/process of submitting the applications so that if camunda is down, still the application submission happens without any interruption.

Now with queues in place, that is not the case anymore. When application is submitted, we just send a message to the queue. And workflow actions service is called when the queue messages are processed, and now we need the queue to fail if camunda is not available to that it can be re-processed when camunda is up.